core/templates/dev/head/pages/static_base.html

```diff
@@ -0,0 +1,238 @@
+      </div>
+    </div>
+
+    <div>
+      <div ng-repeat="message in AlertsService.messages track by $index">
+        <alert-message message-object="message" message-index="$index"></alert-message>
+      </div>
+    </div>
+
+    <div ng-show="loadingMessage" class="oppia-loading-fullpage">
+      <div class="oppia-align-center">
+        <span translate="<[loadingMessage]>"></span>
+        <span class="oppia-loading-dot-one">.</span>
+        <span class="oppia-loading-dot-two">.</span>
+        <span class="oppia-loading-dot-three">.</span>
+      </div>
+    </div>
+    <div ng-show="!loadingMessage">
+    	
+    </div>
+  </div>
+{%- endmacro %}
+
+<!DOCTYPE html>
+<html ng-app="oppia" lang="<[currentLang]>" ng-controller="Base" itemscope itemtype="http://schema.org/Organization">
+  <head>
+    {% block prerender %}
+    {% endblock prerender %}
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
+    <meta name="referrer" content="no-referrer">
+
+    <!-- Tiles for Internet Explorer. -->
+    <meta name="application-name" content="<[siteName]>">
+    <meta name="msapplication-TileColor" content="#ffffff">
+    <meta name="msapplication-square70x70logo" content="{{get_complete_static_resource_url(DOMAIN_URL, '/assets/images/logo/msapplication-tiny.png')}}">
+    <meta name="msapplication-square150x150logo" content="{{get_complete_static_resource_url(DOMAIN_URL, '/assets/images/logo/msapplication-square.png')}}">
+    <meta name="msapplication-wide310x150logo" content="{{get_complete_static_resource_url(DOMAIN_URL, '/assets/images/logo/msapplication-wide.png')}}">
+    <meta name="msapplication-square310x310logo" content="{{get_complete_static_resource_url(DOMAIN_URL, '/assets/images/logo/msapplication-large.png')}}">
+
+    <!-- The itemprops are for G+ sharing. -->
+    <meta itemprop="name" content="{{meta_name}}">
+    <meta itemprop="description" content="{{meta_description}}">
+    <!-- The og tags are for Facebook sharing. -->
+    <meta property="og:title" content="{{meta_name}}">
+    <meta property="og:site_name" content="Oppia">
+    <meta property="og:url" content="{{FULL_URL}}">
+    <meta property="og:description" content="{{meta_description}}">
+    <meta property="og:type" content="article">
+    <meta property="og:image" content="{{get_complete_static_resource_url(DOMAIN_URL, '/assets/images/logo/288x288_logo_mint.png')}}">
+
+    <link rel="apple-touch-icon" href="/assets/images/logo/favicon.png">
+
+    <!-- The title is bound to the rootScope. The content of the block
+    maintitle can be a string or a translation id. If it is a translation it
+    will be replaced by its translation when the page is loading. If it is a
+    string it would be displayed as is. This is the only way to translate
+    the page title because the head of the file is outside the scope of
+    any other controller. -->
+    <div ng-switch="getCurrentUrl()">
```

Comment:
title is served using ng-switch.

I was not able to call function in `translate` and this looked an okayish approach

Reply:
We're getting rid of translate here btw, in case it makes things simpler. (Having issues with wrong snippets showing up in search engines -- see #5647.)

The PR is still on my plate to sort out, but just wanted to let you know.

Reply:
Yes, once that PR is merged, we can handle title similar to other blocks.